### DISCUSSION TITLE
Meeting: March 19th, 2024 agenda

### DISCUSSION THREAD

**Discussion by earth2marsh** [OAI/sig-moonwalk#88] [Announcements]
**Time:** 5pm GMT / 9am Pacific
**Zoom:** [https://zoom.us/j/975841675](https://zoom.us/j/975841675?pwd=SUh4MjRLaEFKNlI3RElpWTdhRDVVUT09), dial-in passcode: 763054

Participants must abide by our [Code-of-Conduct](https://github.com/OAI/OpenAPI-Specification/blob/main/CODE_OF_CONDUCT.md#code-of-conduct).

![image](https://github.com/OAI/sig-moonwalk/assets/54582/09ac7e12-59cd-4061-a7ec-1e36bb6387a7)

| Topic | Owner | Decision/NextStep |
|-------|---------|---------|  
| | | |
Intros and governance meta-topics (5 mins) | SIG members | -- |
General discussion topics (list below) | SIG members | -- |
Can we make links successful? | SIG members | |
Identity location ADR (add link) | SIG members | |

**Comment by handrews** [comment]
I've proposed a bit of repository re-organization and process documentation now that we have a process for making and documenting decisions:

* #89 

HTTP version support advertising (just the version, not any other details discussed) might be simple enough to move to an ADR given the use case details provided here:

* https://github.com/OAI/sig-moonwalk/discussions/45#discussioncomment-8791212

We should pick up @hudlow's topic on the intersection of resource-oriented and schema model variations (e.g. the interaction of read-only, write-only, auto-create-then-read-only, etc. with different operations) since we did not get back to it today.  It is at the intersection of:

* #30 
* #56

**Comment by earth2marsh** [comment]
Thanks everyone for the discussion today! [Video of the meeting](https://zoom.us/rec/play/5aHiasxPBdXUFvW1wuzDm0XptmUsFdzIAWrHmQMGPld7BOdX5gE5PnG-0w97bJ0_2iErr-lqY7-aJyXM.75cDjgvZzFBAfrzX
) is available. 

Some highlights:

- Reviewed and merged #89  @handrews (agreed that until the contributing guidelines in the readme get unwieldy, they can stay there)


- [x] @handrews  close existing issues that should be discussions
- [x] @earth2marsh will set up a tracking issue for a team of maintainers for this SIG. For now that generally maps to regular participants, and we can revisit later.
- [ ]  @darrelmiller will start a governance docs for publishing workflows that we might pioneer here and potentially port over to the 3.x repo.
- [x] Dan gave a quick overview of the topic mentioned above so we can dig in next week (@earth2marsh to copy to next week's agenda)

Please add anything I may have missed.
